dir.props

```diff
@@ -276,7 +276,8 @@
     <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)</SymbolPackageOutputPath>
     <PackageSymbolsBinDir Condition="'$(PackageSymbolsBinDir)' == ''">$(PackagesBasePath)symbols/</PackageSymbolsBinDir>
 
-    <SharedFrameworkPublishDir>$(IntermediateOutputRootPath)sharedFrameworkPublish\</SharedFrameworkPublishDir>
+    <!-- Use / instead of \: used directly by WiX which doesn't accept trailing \. -->
```

Comment:
This comment is a little confusing IMO.  Is the ending period punctuation or the trailing character sequence?  I assume the former.  If this is true consider dropping the punctuation and/or quoting the characters.

```suggestion
    <!-- Use '/' instead of '\': used directly by WiX which doesn't accept trailing '\'. -->
```

Reply:
Good call, your assumption is correct and I prefer the quoting.